packages/difftastic.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Difftastic < Package
+  description 'Difftastic is a structural diff tool that compares files based on their syntax.'
+  homepage 'https://github.com/Wilfred/difftastic'
+  version '0.52.0'
+  license 'MIT'
+  compatibility 'x86_64 aarch64 armv7l'
```

Comment:
This is a terminal app, does it  it work on i686?

Reply:
It might work but is it really worth the effort?